lib/server/server-init.js

```diff
@@ -5,10 +5,11 @@ const getRouter = require('./server-router')
 
 const initRunner = async (app, options = {}) => {
   const {
-    serviceSources,
-    locals
+    serviceSources
   } = await configureSources(options)
 
+  const locals = {govuk_frontend_version: '3.1.0'}
```

Comment:
Odd that we have to manually change this :(

Reply:
i'll have a look to see if it can be removed. then we just use whatever comes thru the package.json

Reply:
Do you know _why_ the app needs this?

Reply:
Apparently, we were typing simultaneously

Reply:
i will leave this. its used to pull in the distribution of the css/js which is found in components core